src/components/listItem.tsx

```diff
@@ -45,7 +45,7 @@ export function ListItem({
   line?: number;
   children?: any[];
 }): JSX.Element {
-  if (checked === null) {
+  if (checked === undefined) {
```

Comment:
`if (checked == null) {` would catch both cases!

Reply:
it's academic, but under what circumstances can `checked` actually be null?